### PR TITLE
Adds Swagger UI for Hub API's

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
# Changes

This patch adds Swagger UI for Hub API's

- Creates a React App Skeleton for Hub Swagger
- Adds and configures eslint and prettier for Hub Swagger
- Creates a config to set the API_URL
- Adds Swagger UI for Hub API
-  Adds Env vars as application config

image

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: swagger/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (pratap0007) [inline swagger/README.md:1]: do we need of readme? if yes , could you please modify it

File: swagger/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (pratap0007) [inline swagger/public/index.html:28]: shoule be  `Hub Swagger` here , WDYT?

File: swagger/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (pratap0007) [inline swagger/public/index.html:5]: we should use tekton icon here as well

File: swagger/package.json [hunk 4]
```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
+    "@testing-library/jest-dom": "^5.11.10",
```
Comment (pratap0007) [inline swagger/package.json:6]: is this change required ?

Comment (PuneetPunamiya) [reply swagger/package.json:6]: I guess since it is updating the library hence it is fine I guess

File: swagger/public/index.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/tekton.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (vinamra28) [inline swagger/public/index.html:10]: can we change the description here?

File: swagger/src/components/App/App.test.tsx [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '.';
+
+test('renders learn react link', () => {
```
Comment (vinamra28) [inline swagger/src/components/App/App.test.tsx:5]: can we change the test description?

File: swagger/public/manifest.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
```
Comment (vinamra28) [inline swagger/public/manifest.json:20]: I guess these can be removed
